__tests__/app.test.js

```diff
@@ -34,3 +48,45 @@ describe("/api/treasures", () => {
     });
   });
 });
+
+describe("/api/articles", () => {
+  describe.only("GET by id", () => {
+    test("200 when request article by valid id", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+    test("200 + returns correct article object with correct properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const article = body.article[0];
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("body");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+        });
+    });
+    test("200 + returns article object with correct id for multiple ids", () => {
+      const reqById = (id) => {
```

Comment:
Sorry this is not the way. In the test we are simulating the actual request, we do not need to get an ID from somewhere else, in the test we are the one choosing the parameters. 
This means that line 76 should look like this:  .get(`/api/articles/1`) exactly how you did it before, and you do not need to do all of the rest, which means that line 80 should look like this:     expect(article.article_id).toBe(1); 

The original test was fine, you do not need to write any other test blocks, only substitute  expect(article).toHaveProperty("article_id"); with expect(article.article_id).toBe(1);  and it will be enough. No need to resubmit, send me a pm on slack